src/libs/isReportMessageAttachment.ts

```diff
@@ -7,15 +7,15 @@ import type {Message} from '@src/types/onyx/ReportAction';
  *
  * @param reportActionMessage report action's message as text, html and translationKey
  */
-export default function isReportMessageAttachment({text, html, translationKey}: Message): boolean {
-    if (!text || !html) {
+export default function isReportMessageAttachment(message: Message | undefined): boolean {
```

Comment:
btw, after the first `if`, you can use like `const {text, html, ...} = message`.

Reply:
I am only able to do this for `text` or `html` fields so I dont know if this is worth it